tutor/src/screens/assignment-edit/details-body.js

```diff
@@ -172,6 +177,10 @@ const DetailsBody = observer(({ ux }) => {
           id="title"
           validate={isRequired}
           data-test-id="edit-assignment-name"
+          
+          innerRef={nameInputField}
+          // Why use 1 and 0 :https://github.com/styled-components/styled-components/issues/1198
+          haserror={!!ux.form.touched.title && !!ux.form.errors.title ? 1 : 0}
         />
```

Comment:
logic looks sound here, but why `1` or `0` instead of `true/false`?   I prefer using `Boolean` to typecast instead of !! since it's more explicit

Seems like this could be: 
`haserror={Boolean(ux.form.touched.title && ux.form.errors.title)}`

Reply:
It was throwing some console errors but the `Boolean()` worked!